albumentations/augmentations/transforms.py

```diff
@@ -3405,3 +3406,65 @@ def get_transform_init_args_names(self):
             "mask_fill_value",
             "random_offset",
         )
+
+
+class FDA(ImageOnlyTransform):
+    """
+    Fourier Domain Adaptation from https://github.com/YanchaoYang/FDA
+    Simple "style transfer".
+    Important: you need to pass target image as a parameter `target_image` in __call__, see example
+
+    Args:
+        beta_limit (float or tuple of float): coefficient beta from paper. Recommended less 0.3.
+
+    Targets:
+        image
+
+    Image types:
+        3-channel uint8 images only
```

Comment:
If we now support grayscale image we can remove this remark.

Reply:
done

Reply:
I think this transform can work with any channel images.